usvm-jvm/src/main/kotlin/org/usvm/api/util/JcTestStateResolver.kt

```diff
@@ -150,7 +199,7 @@ abstract class JcTestStateResolver<T>(
             model.typeStreamOf(ref)
         } else {
             // allocated object
-            memory.typeStreamOf(ref)
+            finalStateMemory.typeStreamOf(ref)
```

Comment:
Why we use `finalStateMemory` here? Maybe it is better to read allocated ref from memory and receive an exception in the case of the model?